### PR TITLE
UI/15399786 dropdown menu

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

**Comment by KatarzynaStefanowiczPagepro** [comment]
https://tw.auroracreation.com/app/tasks/15399786
Dropdown with routing in side navigation

File: src/components/common/Extensions/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -7,15 +8,18 @@ const displayName = 'common.Extensions.Navigation';
 
 const Navigation: FC = () => {
   const isMobile = useMobile();
+  const navigationItems = useSideNavigation();
```
Comment (krawczykkatarzyna) [inline src/components/common/Extensions/Navigation/Navigation.tsx:11]: please move logic to hooks.ts file

File: src/components/common/Extensions/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -7,15 +8,18 @@ const displayName = 'common.Extensions.Navigation';
...
+  const { pathname } = useLocation();
+
+  const currentPage = navigationItems.filter((element) => element.linkTo === pathname);
```
Comment (krawczykkatarzyna) [inline src/components/common/Extensions/Navigation/Navigation.tsx:14]: maybe getCurrentPage will be better name

**Review by krawczykkatarzyna — APPROVED** [review]

File: src/components/common/Extensions/Navigation/Navigation.tsx (hunk 2 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/Navigation/Navigation.tsx:14]: It's not a function, I've changed it to currentPathname

File: src/components/common/Extensions/Navigation/Navigation.tsx (hunk 1 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/Navigation/Navigation.tsx:11]: I use this variable in return

**Review by CzarekDryl — APPROVED** [review]

**Pull request merged**